Makefile

```diff
@@ -171,12 +149,11 @@ cleanup-e2e:
 # To run just one of the e2e tests by name (i.e. 'make TestCreateSimplePipeline'):
 Test%:
 	$(MAKE) cleanup-e2e
-	$(MAKE) deploy-nats
-	$(MAKE) deploy-kafka
-	$(MAKE) deploy-redis
 	$(MAKE) image e2eapi-image
 	kubectl -n numaflow-system delete po -lapp.kubernetes.io/component=controller-manager,app.kubernetes.io/part-of=numaflow
 	kubectl -n numaflow-system delete po e2e-api-pod  --ignore-not-found=true
+	go generate $(shell find ./test/kafka-e2e -name '*.go')
```

Comment:
Why don't use `go generate $(shell find ./test/$* -name '*.go')` as above?

Reply:
i think because that portion of the makefile is `To run just one of the e2e tests by name (i.e. 'make TestCreateSimplePipeline'):` so i dont think `go generate $(shell find ./test/$* -name '*.go')` works as this requires test folder-name as input not a test name.

Reply:
We can try to do a selective go generate here as well by trying to extract the file name 
`shell grep $(*) -R ./test | head -1 | awk -F\/ '{print $$3}'`